stairway/src/test/java/bio/terra/stairway/FlightMapTest.java

```diff
@@ -153,4 +188,48 @@ public void serDesExceptions() {
     Assertions.assertThrows(
         JsonConversionException.class, () -> FlightMap.create(new ArrayList<>(), "garbage"));
   }
+
+  @Test
+  public void guavaTypes() {
+    FlightMap flightMap = new FlightMap();
+
+    final String key = "immutableList";
+    ImmutableList<Integer> immutableListIn = ImmutableList.of(1, 2, 3, 4);
```

Comment:
I'm curious as to why you'd use a guava type for an immutable list rather than use a JDK type.

The guava classes were great when the JDK was lacking features, but now that the JDK has (somewhat) caught up I think it's better to avoid adding a dependency if possible.

Reply:
Yeah, Dan and I were on the fence about this.  There were a few places in WSM that were using `ImmutableList` in what appeared to be test code only, I leaned towards keeping it in as a hedge so as not to have to rev again if that turned out to be wrong.  I could be convinced to back it out though.

Reply:
I don't think the TDR code will need this support, but I can understand stairway wanting to support other possible use cases. When custom serializer support is added this will no longer be your problem. 😀

Reply:
I think the addition of guava is safe and defends against any existing guava use.

With the recent addition of `getRaw` and `putRaw` you can now serialize/deserialize to your heart's content without Stairway messing with your data.